### PR TITLE
Remove more positive test checks (EXPOSUREAPP-12114)

### PR DESCRIPTION
**Pull request by mtwalli**
- Remove more positive test check that were preventing Risk calculation from running
 - Please see my resolved comments for indiction 

## Ticket 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12114

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt [hunk 1]
```diff
@@ -41,16 +41,14 @@ class DeadmanNotificationScheduler @Inject constructor(
...
-                }
+        }.onEach { shouldSchedulePeriodic ->
+            Timber.d("shouldSchedulePeriodic: $shouldSchedulePeriodic")
+            if (shouldSchedulePeriodic) {
+                schedulePeriodic()
+            } else {
+                cancelScheduledWork()
             }
-            .launchIn(appScope)
+        }.launchIn(appScope)
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt:51]: in this file , change are just formatting

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt [hunk 2]
```diff
@@ -38,21 +35,17 @@ class PresenceTracingRiskWorkScheduler @Inject constructor(
...
+                    "isPresenceTracingOnboarded=$isPresenceTracingOnboarded"
             )
-            isAutoMode && isPresenceTracingOnboarded && isRiskCalculationNecessesary
-        }
-            .onEach { runPeriodicWorker ->
-                Timber.tag(TAG).v("runPeriodicWorker=$runPeriodicWorker")
-                setPeriodicRiskCalculation(enabled = runPeriodicWorker)
-            }
-            .launchIn(appScope)
+            isAutoMode && isPresenceTracingOnboarded
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt:44]: core change

Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt:44]: core change

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt [hunk 3]
```diff
@@ -38,21 +35,17 @@ class PresenceTracingRiskWorkScheduler @Inject constructor(
...
-        ) { isAutoMode, isPresenceTracingOnboarded, isRiskCalculationNecessesary ->
+            presenceTracingSettings.isOnboardingDoneFlow
+        ) { isAutoMode, isPresenceTracingOnboarded ->
             Timber.tag(TAG).d(
                 "isAutoMode=$isAutoMode, " +
-                    "isPresenceTracingOnboarded=$isPresenceTracingOnboarded, " +
-                    "isRiskCalculationNecessesary=$isRiskCalculationNecessesary"
+                    "isPresenceTracingOnboarded=$isPresenceTracingOnboarded"
             )
-            isAutoMode && isPresenceTracingOnboarded && isRiskCalculationNecessesary
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt:49]: core change

Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingRiskWorkScheduler.kt:49]: core change

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/execution/ExposureWindowRiskWorkScheduler.kt [hunk 4]
```diff
@@ -45,22 +42,18 @@ class ExposureWindowRiskWorkScheduler @Inject constructor(
...
+                    "isTracing=$isTracing"
             )
-            isAutoMode && isOnboarded && isTracing && isRiskCalculationNecessesary
-        }
-            .onEach { runPeriodicWorker ->
-                Timber.tag(TAG).v("runPeriodicWorker=$runPeriodicWorker")
-                setPeriodicRiskCalculation(enabled = runPeriodicWorker)
-            }
-            .launchIn(appScope)
+            isAutoMode && isOnboarded && isTracing
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/execution/ExposureWindowRiskWorkScheduler.kt:52]: same here

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt [hunk 5]
```diff
@@ -22,35 +21,29 @@ class DeadmanNotificationScheduler @Inject constructor(
...
             enfClient.isTracingEnabled
-        ) { isOnboarded, coronaTests, isTracingEnabled ->
-            val noPositiveTestRegistered = coronaTests.none { it.isPositive }
+        ) { isOnboarded, isTracingEnabled ->
             Timber.d(
                 "isOnboarded = $isOnboarded, " +
-                    "noPositiveTestRegistered = $noPositiveTestRegistered, " +
                     "isTracingEnabled = $isTracingEnabled"
             )
-            isOnboarded && noPositiveTestRegistered && isTracingEnabled
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt:43]: core change

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt [hunk 6]
```diff
@@ -22,35 +21,29 @@ class DeadmanNotificationScheduler @Inject constructor(
...
-        }
-            .onEach { shouldSchedulePeriodic ->
-                Timber.d("shouldSchedulePeriodic: $shouldSchedulePeriodic")
-                if (shouldSchedulePeriodic) {
-                    schedulePeriodic()
-                } else {
-                    cancelScheduledWork()
-                }
+            isOnboarded && isTracingEnabled
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationScheduler.kt:38]: core change

**Review by BMItr — APPROVED** [review]
LGTM!

**Pull request merged**